### DISCUSSION TITLE
付费模式 | shion

### DISCUSSION THREAD

**Discussion by giscus** [shion-app/shion#52] [Announcements]
# 付费模式 | shion

时间追踪软件

https://shion.app/zh/chat/payment-model

**Comment by Ricardolee-Hu** [comment]
把付费版本放在STEAM似乎不是一个好方法，不知道是不是我使用的问题，每次开启都需要先打开steam，相当于steam一直在运行，会白白增加资源消耗，同时也会造成时间记录的不准确吧，毕竟开着steam的时候，我也没打游戏

**Reply by hanaTsuk1** [reply]
`设置`-`开机启动`

并不是打开软件就会计时，见[这里](https://shion.app/zh/guide/collection#monitor)

**Reply by Ricardolee-Hu** [reply]
但是Steam一直开启运行确实很耗费资源呀

**Reply by hanaTsuk1** [reply]
设置开机启动后，**不会**打开steam

如果你就是要手动打开软件，又不想打开steam
在steam仓库找到shion，然后右键，打开所在文件夹，找到对应exe文件，然后创建快捷方式到桌面

**Reply by Ricardolee-Hu** [reply]
好的，我试一试，建议还是在其他平台做付费吧，毕竟我想在办公电脑上用，但是公司电脑装steam也不合适，像我前司，IT策略就是直接 禁止安装的

**Reply by hanaTsuk1** [reply]
好的，我会研究一下的

希望你已经知道了这是个离线应用了，目前还没有让两台机器同步数据的功能，所以办公你其实可以用开源版

**Reply by RicardoDlee** [reply]
嗯嗯，我并不太需要两个设备之间同步，刚好也是想咨询您这个问题，现在有办法绕开steam使用付费版么，倒不是为了数据同步，就是想使用完整版功能。

**Reply by hanaTsuk1** [reply]
那我就把话说得更直白点吧  我没做加密验证  也就是说你可以直接把整个文件夹压缩带走
没有steam后续版本更新不了  情况就是这样

**Reply by RicardoDlee** [reply]
哈哈哈，大佬大气，明白了，仅个人使用，不会作其他用途的

**Comment by RicardoDlee** [comment]
大佬，请问现在界面中的很多模块，特别总览页面加的各种图表都会有一个复选框，但是好像勾选也不会有什么对应的功能，这个是有什么用呢？还是为后面的某些功能预留的？

**Reply by hanaTsuk1** [reply]
多选删除用的  点了之后左下角按钮增加删除一栏

有疑问的话，可以在`discussions`开新主题
